drivers/modem/quectel-bg9x.c

```diff
@@ -1024,13 +1083,11 @@ static int modem_setup(void)
 				       &mdata.rssi_query_work,
 				       K_SECONDS(RSSI_TIMEOUT_SECS));
 
-	/* Once the network is ready, activate PDP context. */
-	ret = modem_cmd_send(&mctx.iface, &mctx.cmd_handler,
-			     NULL, 0U, "AT+QIACT=1", &mdata.sem_response,
-			     MDM_CMD_TIMEOUT);
-
-	/* Retry or Possibly Exit. */
+	/* Once the network is ready, we try to open the PDP context. */
+	ret = modem_pdp_context_open(pdp_contextID);
```

Comment:
As I said in previous review, I think we should use "activate" OR "open". I prefer to stick with "activate" in every place.